testsuite/integration-arquillian/pom.xml

```diff
@@ -578,7 +578,7 @@
         <profile>
             <id>db-oracle</id>
             <properties>
-                <docker.database.image>docker.io/gvenzl/oracle-xe:21-slim-faststart</docker.database.image>
+                <docker.database.image>docker.io/miquelsi/oracle-19c:${oracledb.version}</docker.database.image>
```

Comment:
Is it the correct image?

Reply:
Yes, given that there is no official image for Oracle products, and we were already relying on private images, we created a public image for version 19c.